migration/idempotent/001-base.sql

```diff
@@ -658,25 +676,29 @@ GRANT EXECUTE ON FUNCTION _prom_catalog.get_new_label_id(text, text) to prom_wri
 --wrapper around jsonb_each_text to give a better row_estimate
 --for labels (10 not 100)
 CREATE OR REPLACE FUNCTION _prom_catalog.label_jsonb_each_text(js jsonb, OUT key text, OUT value text)
- RETURNS SETOF record
- LANGUAGE INTERNAL
- IMMUTABLE PARALLEL SAFE STRICT ROWS 10
+    RETURNS SETOF record
+    LANGUAGE INTERNAL
+    SET search_path = pg_catalog
```

Comment:
inlining only applies to sql functions where the body of the sql function gets injected into the actual query. language internal is a C function so it will never get inlined. I dont see the harm in setting search_path explicitly but i'm not sure how much value it actually has for C functions.

Reply:
done